helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml

```diff
@@ -64,6 +64,9 @@ spec:
               value: {{ .Values.timezone | default "UTC" }}
             {{- end }}
           securityContext:
+            {{- if .Values.chaosDaemon.privileged }}
+            privileged: true
+            {{- else }}
             capabilities:
```

Comment:
do we need to remove these capabilities when privileged is true

Reply:
Sorry, we should keep some capabilities.

Reply:
`SYS_PTRACE` must be kept and others can be removed.

Reply:
Other capabilities are unnecessary when privileged is true, except `SYS_PTRACE`. I think we can delete them to make the code cleaner